pipelines.md

```diff
@@ -61,6 +61,10 @@ This task validates the stack is allowed to build and deploy on the cluster.  It
 
    `Deploy-task` uses the `app-deploy.yaml` file to deploy the application to the cluster by using the application deployment operator. By default, the pipelines run and deploy the application in the `kabanero` namespace. If you want to deploy the application in a different namespace, update the `app-deploy.yaml` file to point to that namespace.
 
+- [dockerimage-name-lowercase-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/dockerimage-name-lowercase-task.yaml)
+
+   This task is needed since build and push of a container image requires its imagename to be lowercased as per the docker requirements. This task takes `docker-imagename` and `docker-imagetag` as input params and it builds the `docker-image` resource URL path with `docker-imagename` as lowercase letters. In the case when the input param `docker-imagename` is empty or not provided it will construct the `docker-image` URL path by using the imagename from the existing `app-deploy.yaml` file. This `app-deploy.yaml` file should be present in the `git-source` project provided as an input to the pipeline.
```

Comment:
This task is needed because Docker requires that a container's imagename be lowercase in order to build and push it. This task inputs the `docker-imagename` and `docker-imagetag` parameters and builds the `docker-image` resource URL path with `docker-imagename` as lowercase letters. In the case when the `docker-imagename` parameter is empty or not provided, the `docker-image` URL path is constructed by using the imagename from the existing `app-deploy.yaml` file. This `app-deploy.yaml` file should be present in the `git-source` project that is provided as an input to the pipeline.

Reply:
"in order to build and push it" could be left off the end of the first sentence